encrypt/encrypt.go

```diff
@@ -114,9 +127,15 @@ func Encrypt(args []string) error {
 
 	log.Infof("Ready to encrypt %d file(s)", len(files))
 
-	// Read the public key to be used for encryption. The private key
-	// matching this public key will be able to decrypt the file.
-	publicKey, err := readPublicKey(*publicKeyFile)
+	// Initialize a c4gh public key specs instance
+	c4ghKeySpecs := keySpecs{
+		rgx:    regexp.MustCompile(`-{5}[A-Z]*\s[4A-Z]*\s[A-Z]*\s[A-Z]*-{5}\n.*\n-{5}[A-Z]*\s[4A-Z]*\s[A-Z]*\s[A-Z]*-{5}`),
```

Comment:
you have these regexes - it might be clearer to just use `-----BEGIN CRYPT4GH PUBLIC KEY-----` as a pattern instead of `-{5}[A-Z]*\s[4A-Z]*\s[A-Z]*\s[A-Z]*-{5}` - or does this need to match other strings too? :)

Reply:
my first thought was to allow some verbosity freedom in case the format of the key is changed (for some reason). but I see how this was unnecessary, so I changed it to something more to the point. Thanks for pointing out!